ui/components/forms/flavors/select/index.scss

```diff
@@ -22,3 +22,35 @@
     }
   }
 }
+
+#{$css-prefix}form-element__select-container {
```

Comment:
forgot the `.`

also, to align with our naming patterns, can you change the class to `.#{$css-prefix}select__container`